style.css

```diff
@@ -0,0 +1,488 @@
+    height: auto;
+    width: 80%;
+    margin-top: 7px;
+}
+
+.label-image .text-label {
+    width: 75%;
+    font-size: 15px;
+    color: #666666;
+}
+
+.article-text {
+    font-size: 20px;
+    color: #333333;
+    margin: auto;
+    width: 40%;
+    text-align: justify;
+    line-height: 1.5;
+    margin-bottom: 15px;
+}
+
+
+.article-writer {
+    font-size: 15px;
+    color: #333333;
+    margin: auto;
+    margin-top: 30px;
+    width: 40%;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+.article-writer a {
+    text-decoration: none;
+    font-weight: bold;
+    color: #333333;
+}
+
+.social-media {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-size: 14px;
+    font-family:Helvetica, sans-serif;
+    color: #333333;
+    margin: auto;
+    width: 40%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.social-icon {
+    width: 30px;
+    height: auto;
+
+}
+
+.story-meta {
+    width: 70%;
+    margin: auto;
+}
+
+.imagefloat {
+    /* position: relative; */
+    margin: auto;
+    width: 70%;
+    text-align: justify;
+}
+
+.imagefloat span {
+    font-weight: bold;
+}
+
+.imagefloat img {
+    width: 100%;
+
+}
+
+.wrapper {
+    margin-top: -75%;
+    display: flex;
+    flex-direction: column;
+
+
+}
+
+.first {
+    margin-bottom: 20%;
+}
+
+
+#g-ai1 {
+    float: left;
+    margin-right: 9%;
+    /* top:18.0556%;left:0.2544%; */
+    width: 173px;
+
+}
+
+#g-ai2 {
+    float: left;
+    margin-right: 9%;
+    /* top:18.0556%;left:25.7029%; */
+    width: 173px
+}
+
+#g-ai3 {
+    float: left;
+    margin-right: 9%;
+    /* top:18.0556%;left:51.1511%; */
+    width: 173px;
+
+}
+
+#g-ai4 {
+    float: left;
+    margin-right: 0;
+    width: 173px;
+    ;
+
+}
+
+#g-ai5 {
+    float: left;
+    margin-right: 9%;
+    width: 250px;
+    ;
+
+}
+
+#g-ai6 {
+    float: left;
+    margin-right: 9%;
+    width: 250px;
+    ;
+
+}
+
+#g-ai7 {
+    float: left;
+    margin-right: 9%;
+    width: 250px;
+    ;
+
+}
+
+#g-ai8 {
+    float: left;
+    margin-right: 0;
+    width: 250px;
+
+}
+
+.img-credit {
+    /* position: absolute; */
+    margin-left: 15%;
+    margin-top: 30px;
+    margin-bottom: 30px;
+
+}
+
+.line {
+    margin-top: 30px;
+    width: 75px;
+    position: absolute;
+    left: 15%;
+    border-bottom: 1px #333333 solid;
+}
+
+.secondarticle {
+    margin-top: 7%
+}
+
+.video {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+.videocaption {
+    width: 40%;
+    margin: auto;
+    margin-top: 0;
+    line-height: 0;
+    color: #666666;
+}
+
+.img2 {
+    display: flex;
+    justify-content: center;
+}
+
+.img2text {
+    width: 40%;
+    margin: auto;
+    color: #666666
+}
+
+.article-version {
+    width: 40%;
+    margin: auto;
+    color: #999999;
+    font-size: 12px;
+}
+
+.article-version a {
+    text-decoration: none;
+    color: #999999;
+    font-size: 12px;
+}
+
+.commentsandsocial {
+    display: flex;
```

Comment:
The default value of `flex-direction` is row, so we don't actually need this line.